### PR TITLE
feat(faceted): add slider facet

### PR DESCRIPTION
**Pull request by ncomont**
**What is the problem this PR is trying to solve?**

We need a new kind of facet : Slider Facet

**What is the chosen solution to this problem?**

Implement it : 
image

Here if its configuration : 

```javascript
{
	properties: {
		attribute: 'invalid',
		initialOperatorOpened: true,
		initialValueOpened: false,
		label: 'Invalid',
		operator: {},
		operators: [],
		type: 'slider',
		icon:  {
			name: 'talend-pie-charts',
			class: 'invalid',
		},
		unit: '%',
		defaultValue: 50,
		errorMessage: 'The value must be between 0 and 100',
	},
	metadata: {
		badgePerFacet: 'N',
		entitiesPerBadge: '1',
		operators: [
			'equals',
			'greaterThan',
			'greaterThanOrEquals',
			'lessThan',
			'lessThanOrEquals',
		],
	},
}
```

**Please check if the PR fulfills these requirements**

* [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
* [x] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
* [ ] Docs have been added / updated (for bug fixes / features)
* [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

**Comment by jare-talend** [comment]
The test fails

File: packages/faceted-search/src/components/Badges/BadgeSlider/BadgeSlider.component.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+	value,
+	category,
+	defaultValue = 0,
+	unit = '',
+	...rest
+}) => {
+	const currentOperators = useMemo(() => operators || createDefaultOperators(t), [operators, t]);
+	const currentOperator = operator || currentOperators[0];
+	const badgeTextId = `${id}-badge-slider`;
+	const labelValue = value && `${value}${unit}`;
```
Comment (jare-talend) [inline packages/faceted-search/src/components/Badges/BadgeSlider/BadgeSlider.component.js:45]: I can't take 0 as value and if my value is 0, i should show 0%

File: packages/faceted-search/src/components/Badges/BadgeSlider/BadgeSlider.scss [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+			}
+
+			:global(.form-group) {
+				padding-top: 0;
+				margin-bottom: 0;
+
+				input {
+					font-size: 3.2rem;
+					font-weight: $font-weight-semi-bold;
+					width: 5rem;
```
Comment (jare-talend) [inline packages/faceted-search/src/components/Badges/BadgeSlider/BadgeSlider.scss:40]: increase the value to show 100 correctly

File: packages/faceted-search/src/components/Badges/BadgeSlider/BadgeSliderForm.component.js [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+	id,
+	onChange,
+	onSubmit,
+	value: initialValue,
+	feature,
+	t,
+	unit,
+	icon,
+	decimal = false,
+	min = 1,
```
Comment (jare-talend) [inline packages/faceted-search/src/components/Badges/BadgeSlider/BadgeSliderForm.component.js:28]: set 0  to default value

**Comment by jare-talend** [comment]
In inventoty when I set a quality badge and reopen the application with the saved url, the value isn't take in account.

File: packages/faceted-search/src/components/Badges/BadgeSlider/BadgeSlider.component.js (hunk 1 above)
Comment (ncomont) [reply packages/faceted-search/src/components/Badges/BadgeSlider/BadgeSlider.component.js:45]: ow, good catch, thanks

**Pull request merged**